transformations/chinese_simplified_traditional_perturbation/transformation.py

```diff
@@ -0,0 +1,77 @@
+import itertools
+import random
+import opencc
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+
+def chinese_simplified_traditional_perturbation(text,
+                          prob,
+                          seed,
+                          max_outputs,
+                          config
+                          ):
+    random.seed(seed)
+
+    perturbed_texts = []
+
+    if(config == 's2t'):
+        converter = opencc.OpenCC('s2t.json')
+    elif (config == 't2s'):
+        converter = opencc.OpenCC('t2s.json')
+    else:
+        print("Configuration not specified")
```

Comment:
Is it intented to only print this here? Maybe you could raise an exception or a warning. Does the perturbation works without configuration?

Reply:
I have changed this now so no longer relevant anymore